CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 1.0.3
+ - Updated Google PubSub client library [#27](https://github.com/logstash-plugins/logstash-output-google_pubsub/pull/27)
```

Comment:
We should also state in the changelog that this plugin is only compatible with logstash >= 8.2.0